src/d3a/d3a_core/sim_results/device_statistics.py

```diff
@@ -157,10 +173,4 @@ def _gather_device_statistics(cls, area: Area, subdict: Dict, flat_result_dict:
                                    {market.time_slot: area.strategy.energy_per_slot_kWh})
             cls._calc_min_max_from_sim_dict(subdict, "production_kWh")
 
-        elif isinstance(area.strategy, CommercialStrategy):
```

Comment:
Do we not need this anymore ? I think that at least in the d3a plots, this block is still needed.

Reply:
Judging from the comments of @sarahhambridge in ticket, where she mentioned `energy buffer is not needed`. So, I have omitted it